attributes/default.rb

```diff
@@ -1,40 +1,74 @@
+
+# Checksum for Ice WAR file.
 node.default['ice']['checksum']                             = 'eb9e7503585553bdebf9d93016bcbe7dc033c21e2b1b2f0df0978ca2968df047'
+
+# HTTP URL for Ice WAR file.
 node.default['ice']['war_url']                              = 'https://s3.amazonaws.com/dl.imedidata.net/ice'
+
+# Will force a redeploy of the Ice WAR file.
 node.default['ice']['force_redeploy']                       = false
 
+# Organization name that is displayed in the UI header.
 node.default['ice']['company_name']                         = nil
 
+# Enables the ice processor.
 node.default['ice']['processor']['enabled']                 = true
-node.default['ice']['processor']['local_dir']                = '/var/ice_processor'
+
+# Local work directory for the Ice processor.
+node.default['ice']['processor']['local_dir']               = '/var/ice_processor'
+
+# Optional. Work around https://github.com/Netflix/ice/issues/100 by pre-creating
+# the directories it expects to find. Default: false
 node.default['ice']['processor']['issue_100_workaround']    = false
 
-# S3 keys for accessing billing files
+# AWS access key id used for accessing AWS detailed billing files from S3.
 node.default['ice']['billing_aws_access_key_id']            = nil
+# AWS secret key used for accessing AWS detailed billing files from S3.
 node.default['ice']['billing_aws_secret_key']               = nil
 
+# Name of the S3 bucket containing the AWS detailed billing files.
 node.default['ice']['billing_s3_bucket_name']               = nil
+# Directory in the S3 billing bucket containing AWS detailed billing files.
 node.default['ice']['billing_s3_bucket_prefix']             = nil
 
-node.default['ice']['billing_payerAccountId']				= nil
-node.default['ice']['billing_accessRoleName']				= nil
-node.default['ice']['billing_accessExternalId']				= nil
+node.default['ice']['billing_payerAccountId']               = nil
+node.default['ice']['billing_accessRoleName']               = nil
+node.default['ice']['billing_accessExternalId']             = nil
 
+# Name of the S3 bucket that Ice uses for processed AWS detailed billing files.
+# This bucket is shared between the Ice processor and reader.
 node.default['ice']['work_s3_bucket_name']                  = nil
+
+# Directory in the S3 work bucket containing the processed AWS detailed billing files.
 node.default['ice']['work_s3_bucket_prefix']                = nil
 
+# Enables the Ice reader and installs the Nginx reverse proxy.
 node.default['ice']['reader']['enabled']                    = true
-node.default['ice']['reader']['local_dir']                   = '/var/ice_reader'
+# Local work directory for the Ice reader.
+node.default['ice']['reader']['local_dir']                  = '/var/ice_reader'
 
-node.default['ice']['start_millis']                          = 0
+# Specify the start time in milliseconds for the processor to start processing billing files.
+node.default['ice']['start_millis']                         = 0
```

Comment:
Explaining that start_millis is milliseconds since start of unix epoch time.  Setting default value to 90 days in the past.